src/test/index.js

```diff
@@ -22,44 +31,91 @@ jest.mock( '@wordpress/react-native-editor/src/setup', () => ( {
 	default: jest.fn().mockReturnValue( <></> ),
 } ) );
 
-const initialProps = { initialData: '', capabilities: {} };
+const defaultLocale = 'en-gb';
+const setupLocaleLogs = [
+	[ 'locale', defaultLocale, getGutenbergTranslation( defaultLocale ) ],
+	[
+		'jetpack - locale',
+		defaultLocale,
+		getJetpackTranslation( defaultLocale ),
+	],
+	[
+		'layout-grid - locale',
+		defaultLocale,
+		getLayoutGridTranslation( defaultLocale ),
+	],
+];
```

Comment:
These logs correspond to the setup locale logs triggered from [here](https://github.com/WordPress/gutenberg/blob/9675b99c6e7c88af16749acc3dd0dcdfbe35db27/packages/react-native-editor/src/setup-locale.js#L36-L44).

Reply:
AFAIK, we don't display too many informational logs, apart from the locale/translations ones, but in any case, I like your idea about providing an option for enabling/disabling logs. As you mentioned, it would help us on identifying important messages like warnings and errors 👍.